src/components/AppButtons.tsx

```diff
@@ -49,26 +49,25 @@ const mapDispatchToProps =
         };
     };
 
-const AppButtons = ({ initHandler, updateNodeColorHandler, addNodeHandler }: AppButtonsDispatchProps) => {
-    return (
+class AppButtons extends React.Component<AppButtonsDispatchProps, {}> {
```

Comment:
there is no reason to replace the stateless functional component (which is a pure component). This component has only a render function.